lntest/timeouts.go

```diff
@@ -23,5 +23,5 @@ const (
 
 	// AsyncBenchmarkTimeout is the timeout used when running the async
 	// payments benchmark.
-	AsyncBenchmarkTimeout = time.Minute
+	AsyncBenchmarkTimeout = 2 * time.Minute
```

Comment:
Was this increase needed for travis or when running this test locally?

Reply:
This was for me locally, not sure if it also happens on Travis. I think this was just too short in any case, we should be able to run the tests on any system, especially locally.